### ISSUE TITLE
Question on outgoing commitment column

### ISSUE THREAD

**Issue by PetyaKangalova** [devinit/D-Portal#629]
I received the below questions from a publisher:

1. On the main page, the data is showing “Outgoing commitments”, but many ITC projects do not have outgoing commitments and they are not shown under “Transactions”. Could you please let us know what is displayed under “Outgoing commitments”?
https://d-portal.org/ctrack.html?publisher=XM-DAC-45001#view=active

![image](https://user-images.githubusercontent.com/24607542/131642763-c952b6f3-431f-42e3-a485-cb8233132ccb.png)

From the Glossary we say:

<html>
<body>

Total Outgoing Commitment | Select the minimum and/or maximum sum of all Outgoing Commitment type transactions.
-- | --

</body>
</html>

It is true that in this example https://d-portal.org/ctrack.html?publisher=XM-DAC-45001#view=act&aid=XM-DAC-45001-B678, there are no outgoing commitments, only **incoming commitment**. @notshi  and @xriss does d-portal take any commitment types for that column?

2. In the CSV file, could you please tell us how the “total-commitment” is calculated?
![image002](https://user-images.githubusercontent.com/24607542/131641534-d433ed72-846d-4d86-85d0-2566df2464be.png)

@notshi  and @xriss  can you confirm what the total commitment is picking up? Is it the sum of both incoming and outgoing commitments or just outgoing? I wasn't able to find in the FAQ.

**Comment by notshi** [comment]
Hi @PetyaKangalova, many thanks for raising this.

1. For most activities, this column displays 'Outgoing Commitments' but rather than have empty data where no Outgoing Commitment (2) are available, it falls back on Spend instead; ie. Disbursement (3) + Expenditure (4).

The numbers in brackets are the `transaction-type @code` ([url](https://iatistandard.org/en/iati-standard/203/codelists/transactiontype/#codes)).

> From the Glossary we say:

The Glossary is only used for the search filters and does not apply to this page. 
This is why the term 'Outgoing Commitment type transactions' is linked to the `transaction-type @code` list.
If you do use that particular outgoing commitment filter on the search page, the search would perform as expected.

The _updated_ labelling of that column on the list of activities page comes from https://github.com/devinit/D-Portal/issues/462
The reasoning behind the bars comes from https://github.com/devinit/D-Portal/issues/434

We could update the the column name and FAQ when that is done so that this is more clear.
If that is the case, what should the label of the column be?

On another note, Spend should probably also include other newer `transaction-type @code` but we are not sure which ones. When this was designed, we were on version 1.04 of the Standard so would appreciate any updated advice.

2. The column, total-commitment is calculated as we have described in the [FAQ](https://d-portal.org/faq.html#tech1) and point 1 above.

> ‘Expenditure’ and ‘Disbursements’ for Spend columns/data. If there are neither, but ‘Commitment’ data exists, then the Commitment data will be shown.

This means that column will display the total of all Outgoing Commitment (2).
If an activity does not have Outgoing Commitment (2), we will display the total of Disbursement (3) + Expenditure (4).

This is why some activities have the same total-commitment and total-spend data in the columns.

We have only used Outgoing Commitment (2) as d-portal was created during version 1.04 of the standard, [before numerical values were added as codes](https://github.com/IATI/IATI-Codelists/blob/version-1.04/xml/TransactionType.xml).

Incoming Commitment (11) was only introduced in version 2.02 which also changed the previous naming of Commitment (C).

The FAQ text was supplied to us by IATI so it would be good to get an updated version of that.

**Comment by PetyaKangalova** [comment]
hi @notshi  thank you  so much for the quick response and  very clear explanations. I'd suggest two things:

1. Change the column heading from "Outgoing Commitment" to " Total Commitment". In that way it is aligned with the CSV and to Michelle's point we can clarify in FAQ that it looks at outgoing commitments only.
2.  Update FAQ as below to explain how "Total commitment" is being calculate. Suggested text for FAQ.

**Which transaction types are used/not used in d-portal?**
‘Expenditure’ and ‘Disbursements’ for Spend columns/data. If there are neither, but ‘Outgoing Commitment’ data exists, then the  Outgoing Commitment data will be shown.
'Outgoing commitment' for Total commitment columns. If outgoing commitments have not been published, but Disbursement  and Expenditure exist, then the sum of Disbursement and Expenditure will be shown.
‘Budgets’ in activity file, for Budget columns/data (eg in tables on d-portal) - if no budget data is present in the activity file, then ‘planned-disbursements’ are used if present.

Finally, to your point about spend and including other transactions, currently in IATI when we refer to spend we only use Disbursements and Expenditures, so no changes there.

**Comment by notshi** [comment]
Many thanks, @PetyaKangalova - these changes are now live on the website.

The column heading has been changed to 'Total Commitment' and the FAQ has been updated with the suggested text.

Have also updated all instances of 'outgoing commitments' to 'total commitments' in the footer text for all pages and hover text for all tables. We've also included information about total commitments in the main page footer under 'Project List' for completeness.

Do let me know if I've missed anything.
